communitiesuk/funding-service-design-assessment-store
# FS-3411 cof-r3w2-config

https://dluhcdigital.atlassian.net/browse/FS-3411
https://dluhcdigital.atlassian.net/browse/FS-3417

### Description
- Mapping config (scored & unscored) added for COF r3w2. Mapping references are [here](https://app.mural.co/t/mhclg2837/m/mhclg2837/1690378945342/0c7733b2180793a6f5b2a206cc0f673c851c6fd5?sender=u0eed0f101f1e77432de57236)
- Import script assessment config added for COF r3w2

### How to test locally
1.  seed the COFR3W2  application
   `docker exec -it <container_id> invoke seed_dev_db --fundround COFR3W2 --appcount 1`
2. open assessment dashboard & notice the seeded application
3. click on the application to see unscored and scored sections

### Screenshots of UI changes (if applicable)
![image](https://github.com/communitiesuk/funding-service-design-assessment-store/assets/127315890/15ddfae8-3b1e-4bf8-967f-fac3464600a6)
